content/efcore/getting-started/toc.md

```diff
@@ -0,0 +1,2 @@
+- Introduction to Entity Framework Core
```

Comment:
These will be the names of the chapters displayed on the sidebar of the website. To avoid them being too long, let's abbreviate Entity Framework:

Intro to EF Core
Adding EF Core to a Project